packages/azure-openapi-validator/autorest/src/spectral-plugin-func.ts

```diff
@@ -212,7 +213,10 @@ async function runSpectral(doc: any, filePath: string, sendMessage: (m: Message)
   return mergedResults
 }
 
-export async function getRuleSet(openapiType: OpenApiTypes): Promise<[ruleset: Ruleset, namesOfRulesInStagingOnly: string[]]> {
+export async function getRuleSet(
+  openapiType: OpenApiTypes,
+  filterForTypespec: boolean
```

Comment:
suggest to rename `filterForTypespec` to `removeRulesIgnoredByTypespec`

Reply:
That's the same name you suggested for the method, so it causes a name conflict. I used `isTypespecGenerated` instead